src/components/SkillCreator/SkillCreator.js

```diff
@@ -1295,6 +1295,7 @@ export default class SkillCreator extends Component {
                               >
                                 <input
                                   type="file"
+                                  accept="image/x-png,image/gif,image/jpeg"
```

Comment:
Make it `image/*` instead for simplicity.

Reply:
Alright!